plugins/typescript/src/core/getRequestBodyType.ts

```diff
@@ -50,6 +50,11 @@ export const getRequestBodyType = ({
     return f.createKeywordTypeNode(ts.SyntaxKind.UndefinedKeyword);
   }
 
+  // NOTE: Maybe not perfect to have this hardcoded here
+  if (mediaType.contentType === "application/x-www-form-urlencoded") {
+    return f.createTypeReferenceNode("URLSearchParams", undefined);
```

Comment:
Sure thing, If would add it to `getOperationTypes.test.ts` right? What kind of test runner are you using?

Reply:
https://github.com/fabien0102/openapi-codegen/blob/main/package.json#L17

Jest 😉

Reply:
For the location of the test, I let you decide where this makes sense. The idea is to be sure this will not be broken later and to see that this is working as expected 😃